Makefile

```diff
@@ -72,6 +69,9 @@ install-docs: docs/skopeo.1
 	install -d -m 0755 ${MANINSTALLDIR}/man1
 	install -m 644 docs/skopeo.1 ${MANINSTALLDIR}/man1/
 
+install-completions:
+	install -m 644 -T hack/make/bash_autocomplete ${BASHINSTALLDIR}/skopeo
```

Comment:
Odd that the path wouldn't already be there.. but I can I can add -D making it:
```
install -m 644 -D -T hack/make/bash_autocomplete ${BASHINSTALLDIR}/skopeo
```
which will create the path first then copy the file with the name change.